README.md

```diff
@@ -23,7 +23,7 @@ Your typical development workflow will look something like this:
 7. **Push** your branch to your forked repository.
 8. **Open a Pull Request** to the original repository.
 
-For more detailed instructions, check out this guide (_TBD_).
+For more detailed instructions, check out this [contribution guide](https://contribute.freecodecamp.org/how-to-setup-freecodecamp-locally/).
```

Comment:
this is the link to the guide on how to set up the freecodecamp.org site locally, not the contribute site

I am not sure there is a guide to contribute to the contribute site

Reply:
Thats correct. I have never gotten around to detailing the instructions on working on the `contribute` repo itself beyond the readme.